modules/cnf-configuring-nrop-on-schedlable-control-planes.adoc

```diff
@@ -0,0 +1,224 @@
+
+This configuration allows the NUMA Resources Operator to schedule pods on both control plane and worker nodes, taking into consideration resource availability and NUMA topology. The NUMA Resources Operator treats Control plane nodes like any other schedulable node on the cluster, and they can be included in the `nodeGroups` for Topology-Aware Scheduling (TAS).
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the NUMA Resources Operator.
+
+.Procedure
+
+. To enable Topology Aware Scheduling (TAS) on control plane nodes, configure the nodes to be schedulable first. This allows the NUMA Resources Operator to deploy and manage pods on them. Without this action, the operator cannot deploy the pods required to gather NUMA topology information from these nodes. Follow these steps to make the control plane nodes schedulable:
+
+.. Edit the `schedulers.config.openshift.io` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit schedulers.config.openshift.io cluster
+----
+
+.. In the editor, set the `mastersSchedulable` field to `true`, then save and exit the editor.
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Scheduler
+metadata:
+  creationTimestamp: "2019-09-10T03:04:05Z"
+  generation: 1
+  name: cluster
+  resourceVersion: "433"
+  selfLink: /apis/config.openshift.io/v1/schedulers/cluster
+  uid: a636d30a-d377-11e9-88d4-0a60097bee62
+spec:
+  mastersSchedulable: true
+status: {}
+#...
+----
+
+. Configure the NUMA Resources Operator to manage the schedulable control plane nodes. The `nodeGroups` configuration in the NUMAResourcesOperator CR tells the Operator which nodes to manage. There must be only one `numaresourcesoperator` resource on the cluster.
+
+.. For a Compact Cluster, the compact cluster's master nodes are also the schedulable nodes, so specify only the master pool. Create the following `nodeGroups` configuration in the `NUMAResourcesOperator` CR:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesOperator
+metadata:
+  name: numaresourcesoperator
+spec:
+  nodeGroups:
+    - poolName: master
+----
++
+[NOTE]
+====
+Using the worker pool in addition to the master pool in a compact cluster can create issues with duplicate pods. 
+====
+
+.. For an MNO custer, since both the control plane and worker nodes are schedulable, specify both the master and worker pools. Create the following `nodeGroups` configuration in the `NUMAResourcesOperator` CR:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesOperator
+metadata:
+  name: numaresourcesoperator
+spec:
+  nodeGroups:
+    - poolName: master
+    - poolName: worker
+----
+
+.. After you create the appropriate `nodeGroups` configuration for your cluster type, apply the changes by running the following command:
```

Comment:
create -> update/specify
the nodeGroups are field in the NRO CR and are pointing to a valid MCPs